### PR TITLE
[shellinabox] Add guide for Shell In A Box

### LINKED ISSUE #466: Add guide for shellinabox
I'm using https://github.com/shellinabox/shellinabox. Would this be a good guide?

### PR DESCRIPTION
**Pull request by lk3de**
Hi, I've written a guide for [shellinabox](https://github.com/shellinabox/shellinabox). 🙂 Shell In A Box emulates a terminal in your browser that works with purely JavaScript and CSS across most modern web browsers.

closes #466

### CODE REVIEW

**Comment by luto** [comment]
uuh, I look forward to reviewing this one :grin: thanks!

File: source/guide_shellinabox.rst [hunk 1]
```diff
@@ -29,78 +29,81 @@ Shell In A Box is released under the `GNU General Public License version 2`_.
...
+To run *shellinabox* as a deamon, create a new file ``~/etc/services.d/shellinabox.ini`` with the following content:
 
-.. warning:: Replace ``isabell`` with your username and ``<port>`` with the port that you have opened in the firewall!
+.. warning:: Replace ``isabell`` with your username!
 
 .. code-block:: ini
 
  [program:shellinabox]
- command=/home/isabell/shellinabox/shellinaboxd --background --disable-ssl --port <port> --verbose
+ command=/home/isabell/shellinabox/shellinaboxd --disable-ssl --verbose
```
Comment (pxlfrk) [inline source/guide_shellinabox.rst:95]: i would recommend using `command=%(ENV_HOME)s/shellinabox/shellinaboxd --disable-ssl --verbose` instead :)

File: source/guide_shellinabox.rst [hunk 2]
```diff
@@ -29,78 +29,81 @@ Shell In A Box is released under the `GNU General Public License version 2`_.
...
+.. note:: *shellinabox* is running on port 4200 by default.
 .. include:: includes/web-backend.rst
 
 Setup daemon
 ------------
-Create a new file ``~/etc/services.d/shellinabox.ini`` with the following content:
+To run *shellinabox* as a deamon, create a new file ``~/etc/services.d/shellinabox.ini`` with the following content:
 
-.. warning:: Replace ``isabell`` with your username and ``<port>`` with the port that you have opened in the firewall!
+.. warning:: Replace ``isabell`` with your username!
```
Comment (pxlfrk) [inline source/guide_shellinabox.rst:90]: not necessary, if you use  `ENV_HOME` below, see my comment.

File: source/guide_shellinabox.rst (hunk 1 above)
Comment (luto) [reply source/guide_shellinabox.rst]: thanks :)

**Comment by luto** [comment]
Thank you! :tada:

**Pull request merged**